doc/unet.txt

```diff
@@ -0,0 +1,154 @@
+.. _unet:
+
+U-Net
+**********************************************
+
+.. note::
+    This section assumes the reader has already read through :doc:`lenet` for
+    convolutional networks motivation and :doc:`fcn_2D_segm` for segmentation
+    network.
+
+Summary
++++++++
+
+This tutorial provides a brief explanation of the U-Net architecture as well as a way to implement
+it using Theano and Lasagne. U-Net is a Fully Convolutional Network (FCN) that does image segmentation.
+Its goal is then to predict each pixel's class. See :doc:`fcn_2D_segm` for differences between
+network architecture for classification and segmentation tasks.
+
+Model
++++++
+
+The U-Net architecture is built upon the Fully Convolutional Network and modified
+in a way that it yields better segmentation in medical imaging.
+Compared to FCN-8, the two main differences are (1) U-net is symmetric and (2) the skip
+connections between the downsampling path and the upsampling path apply a concatenation
+operator instead of a sum. These skip connections intend to provide local information
+to the global information while upsampling.
+Because of its symmetry, the network has a large number of feature maps in the upsampling
+path, which allows to transfer information. By comparison, the basic FCN architecture only had
+*number of classes* feature maps in its upsampling path.
+
+The U-Net owes its name to its symmetric shape, which is different from other FCN variants.
+
+U-Net architecture is separated in 3 parts:
+
+- 1 : The contracting/downsampling path
+- 2: Bottleneck
+- 2 : The expanding/upsampling path
```

Comment:
I guess the automatic rendering makes it a "3", but it is a bit confusing to read the source :)